src/components/reports/ReportPA.vue

```diff
@@ -0,0 +1,373 @@
+  },
+};
+
+const skillFocusByGradePA = {
+  description: "Skills to Focus On by Grade Level",
+  title: {
+    text: "Skills to Focus On",
+    subtitle: "by grade level",
+    anchor: "middle",
+    fontSize: 18,
+  },
+  height: 400,
+  width: 600,
+  data: { values: scoreStore.scores },
+  transform: [
+      // convert mastery to focus so we can use sum to count the number who need to work on the skill
+      {calculate: "(datum.masteryFSM==0)?1:0", as: "FSM"},
+      {calculate: "(datum.masteryLSM==0)?1:0", as: "LSM"},
+      {calculate: "(datum.masteryDEL==0)?1:0", as: "DEL"},
+  ],
+  "repeat": {"layer": ["FSM", "LSM", "DEL"]},
+  "spec": {
+    "mark": "bar",
+    "encoding": {
+
+      "y": {
+        "field": "grade",
+        "type": "nominal",
+        axis: {labelAngle:"0"},
+      },
+      "x": {
+        "aggregate": "sum",
+        "field": {"repeat": "layer"},
+        "type": "quantitative",
+        "title": "# of students"
+      },
+      "color": {
+        "datum": {"repeat": "layer"}, 
+        "title": "Skill",
+        scale: {range: ["#be95c4","#5e548e", "#0072b2", ]},
+      },
+      "yOffset": {"datum": {"repeat": "layer"}}
+    }
+  },
+  "config": {
+    "mark": {"invalid": null}
+  }
+};
+
+const normedPercentileDistribution1to4PA = {
+  // ...globalChartConfig,
+  description: "Distribution of Normed Percentiles (all grades)",
+  title: {
+    text: "Distribution of CTOPP Equivalent Percentiles",
+    anchor: "middle",
+    fontSize: 18,
+    subtitle: "(grades K to 4)",
+    subtitleFontStyle: "bold",
+    subtitleFontSize: 12,
+  },
+  height: 200,
+  width: 600,
+  data: { values: scoreStore.scores },
+
+  transform: [
+    {  calculate: "datum.percentileRankCTOPP <= 25? 'Extra Support Needed': datum.percentileRankCTOPP <=50? 'Some Support Needed': 'Average or Above Average' ",
+       as: "SupportGrade1to4",
+    },
+    {  calculate: "datum.percentileRankCTOPP <= 15? 'Extra Support Needed': datum.percentileRankCTOPP <=30? 'Some Support Needed': 'Average or Above Average' ",
+       as: "SupportGrade5to12",
+    },
+    {"filter": "(datum.grade == 'Kindergarten') || (datum.grade <= 4)" },  
+  ],
+
+  mark: "bar",
+  encoding: {
+    x: {
+      bin: true,
+      field: "percentileRankCTOPP",
+      title: "Percentile (relative to national norms)",
+      scale: { domain: [0, 100] },
+      bin: { step: 5, "extent":[0,100], },
+      axis: { tickMinStep: 1 },
+    },
+    y: { aggregate: "count", title: "count of students" },
+    order: { field: "order" },
+    color: {
+      field: "SupportGrade1to4",
+      title: "Support",
+      scale: {
+        domain: ["Extra Support Needed", "Some Support Needed", "Average or Above Average"],
+        //domain: ["At Risk", "Some Risk", "Doing Well"],
+        range: ["#cc79a7", "#f0e442", "#0072b2"],
+      },
+      order: { field: "order", type: "nominal" },
+    },
+  },
+};
+
+const normedPercentileDistribution5to12PA = {
+  // ...globalChartConfig,
```

Comment:
If these instances of globalChartConfig are not relevant anymore, let's delete them before going to main